src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -273,6 +278,58 @@ EmberAfNetworkCommissioningError OnEnableNetworkCommandCallbackInternal(app::Com
     return err;
 }
 
+static void ThreadScanNetworksResultCallback(chip::DeviceLayer::ConnectivityManager::ThreadDiscoveryResult * result)
+{
+    if (result == nullptr)
```

Comment:
nit: the function might take a reference instead of a pointer in which case the null-check wouldn't bee needed.